checker/src/org/checkerframework/checker/fenum/qual/FenumBottom.java

```diff
@@ -1,14 +1,18 @@
 package org.checkerframework.checker.fenum.qual;
 
-import java.lang.annotation.*;
-
-import com.sun.source.tree.Tree;
-
 import org.checkerframework.framework.qual.DefaultFor;
-import org.checkerframework.framework.qual.TypeUseLocation;
 import org.checkerframework.framework.qual.ImplicitFor;
 import org.checkerframework.framework.qual.SubtypeOf;
 import org.checkerframework.framework.qual.TargetLocations;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+import java.lang.annotation.Documented;
```

Comment:
I followed the order of imports @wmdietl  uses in Eclipse which he exported and stored in [checker/eclipse/checker-framework.importorder](
https://github.com/typetools/checker-framework/blob/master/checker/eclipse/checker-framework.importorder)  

I don't have an opinion on the order of imports.  I just want to configure IntelliJ in such a way that imports on files under code review do not distract the reviewers.